docs/design.md

```diff
@@ -149,7 +149,8 @@ The `Logic` component handles the business logic of TEAMMATES. In particular, it
 
 Package overview:
 - **`logic.api`**: Provides the API of the component to be accessed by the UI.
-- **`logic.core`**: Contains the core logic of the system.
+- **`logic.core`**: Contains the core logic of the system, mainly CRUD of storage entities.
+- **`logic.external`**: Holds the logic of external services such as GAE task queue service.
```

Comment:
"GAE task queue service" doesn't exist (or rather, is not what we use). If you want to use this as example, just say task queue service without any specific vendor name.

Reply:
Thanks. Changes are made accordingly.